admin/notifiers/class-configuration-notifier.php

```diff
@@ -0,0 +1,175 @@
+<?php
+/**
+ * WPSEO plugin file.
+ *
+ * @package WPSEO\Admin\Notifiers
+ */
+
+/**
+ * Represents the logic for showing the notification.
+ */
+class WPSEO_Configuration_Notifier implements WPSEO_Listener {
+
+	/**
+	 * Option name used to determine whether the notice has been dismissed.
+	 *
+	 * @var string
+	 */
+	const META_NAME = 'wpseo-dismiss-configuration-notice';
+
+	/**
+	 * Default value.
+	 *
+	 * @var string
+	 */
+	const META_VALUE = 'yes';
```

Comment:
Depending on feedback on whether the notification should be dismissible, this might be not needed.